ra/ra.go

```diff
@@ -1953,6 +1953,10 @@ func (ra *RegistrationAuthorityImpl) checkOrderNames(names []string) error {
 
 // NewOrder creates a new order object
 func (ra *RegistrationAuthorityImpl) NewOrder(ctx context.Context, req *rapb.NewOrderRequest) (*corepb.Order, error) {
+	if req == nil || req.RegistrationID == 0 {
+		return nil, errors.New("incomplete gRPC request message")
```

Comment:
If this lands after #5478, then it should incorporate the file-level `errIncompleteGRPCRequest` introduced there.

Reply:
done